tests/lib/ringbuffer/testcase.yaml

```diff
@@ -1,6 +1,6 @@
 common:
     tags: ring_buffer circular_buffer
-
+    filter: not CONFIG_SMP
```

Comment:
Could you please just force platform to run on one cpu core (by setting `CONFIG_MP_NUM_CPUS=1`) instead of filter out test?